test/docker/compose.go

```diff
@@ -265,7 +265,7 @@ func (d *Compose) WithText2VecVoyageAI() *Compose {
 }
 
 func (d *Compose) WithText2VecPaLM(apiKey string) *Compose {
-	d.withPaLMApiKey = apiKey
+	d.withGoogleApiKey = apiKey
```

Comment:
wondering if the method name should be changed just for consistency to `WithText2VecGoogle`

Reply:
yes, it will be changed in this [PR](https://github.com/weaviate/weaviate/pull/5816) (and another great catch!)